frontend/src/app/PollRoutes.tsx

```diff
@@ -81,7 +88,7 @@ const PollRoutes = ({ pollName }: Props) => {
           return;
         }
         return (
-          <route.type key={route.id} path={`${basePath}/${route.url}`}>
+          <route.type key={route.id} path={`${basePath}/${route.url}`} exact>
```

Comment:
At first, I was concerned that using `exact` could break some existing URLs, but it seems to be fine. And react-router will route URLs with a trailing slash to the same route. :+1:

Reply:
Si, I added exact to make the routing independent of the routes declaration order. For instance, it prevents the page `/presidentielle2022/comparison` to match the home page `/presidentielle2022/`.

If at some point we experience unexpected behaviours we could always declare this prop conditionally.